sdk/index.js

```diff
@@ -23,10 +23,7 @@
 module.exports.AkkaServerless = require('./src/akkaserverless').AkkaServerless;
 module.exports.EventSourcedEntity = require('./src/event-sourced-entity');
 module.exports.ValueEntity = require('./src/value-entity');
-module.exports.ReplicatedEntity =
-  require('./src/replicated-entity').ReplicatedEntity;
-module.exports.ReplicatedData =
-  require('./src/replicated-entity').ReplicatedData;
+module.exports.replicatedentity = require('./src/replicated-entity');
```

Comment:
Typing for `ReplicatedEntity` didn't show up unless this mirrored the namespace structure.